src/src/SelfServiceApiClient.js

```diff
@@ -587,6 +599,31 @@ export class SelfServiceApiClient {
       );
     }
   }
+
+  CheckCanBypassMembershipApproval(capabilityDefinition) {
+    const link = capabilityDefinition?._links?.joinCapability;
```

Comment:
'join' and 'bypassMembershipApplication' does not feel like they relate.
I know they do, but it feels a bit strange